getting-started/kubernetes/minikube.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Quickstart for Calico on minikube
+description: Enable Calico on a single/multi-node minikube cluster for testing or development in under 1 minutes.
+canonical_url: '/getting-started/kubernetes/minikube'
+---
+
+### Big picture
+
+This quickstart gets you a single-node minikube cluster with {{site.prodname}}
+in approximately 1 minutes. You can use this cluster for testing and
+development.
+
+### Value
+
+Use this quickstart to quickly and easily try {{site.prodname}} features.
+
+### Before you begin
+
+There are few requirements that you will need if you like to follow along with this document.
+
+requirements:
```

Comment:
I don't think you need to say "requirements:" or the previous sentence. The title of "Before you begin" is enough in this case.